src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml

```diff
@@ -49,9 +49,38 @@ let plugin_flag =
          times"
   else Command.Param.return []
 
+let make_ledger_backing ~logger ~constraint_constants ~runtime_config
+    ~hardfork_handling =
+  let current_genesis_global_slot =
+    let%map.Option { global_slot_since_genesis = current_genesis_global_slot
+                   ; _
+                   } =
+      constraint_constants.Genesis_constants.Constraint_constants.fork
+    in
+    current_genesis_global_slot
+  in
+  let hardfork_slot =
+    let%map.Option hardfork_slot_since_last_hf =
+      Runtime_config.scheduled_hard_fork_genesis_slot runtime_config
+    in
+    Mina_numbers.Global_slot_since_hard_fork.to_global_slot_since_genesis
+      ~current_genesis_global_slot hardfork_slot_since_last_hf
+  in
+
+  match (hardfork_handling, hardfork_slot) with
+  | Cli_lib.Arg_type.Hardfork_handling.Migrate_exit, Some hardfork_slot ->
+      Mina_ledger.Root.Config.Converting_db hardfork_slot
+  | Migrate_exit, _ ->
+      failwith "No hardfork slot provided for Migrate_exit mode"
```

Comment:
I think this message might be a bit more helpful if it explains that both `daemon.slot_chain_end` and `daemon.hard_fork_genesis_slot_delta` must be set for `Migrate_exit` to be used.